docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -93,6 +93,8 @@ If `true`, the snapshot's persistent settings, index templates, ingest
 pipelines, and {ilm-init} policies are restored into the current cluster. This
 overwrites any existing cluster settings, templates, pipelines and {ilm-init}
 policies whose names match those in the snapshot.
+<2> Regular expression that matches data streams and indices in the snapshot. Supports https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html-[regular-expression constructs].
```

Comment:
```suggestion
<2> Regular expression that matches data streams and indices in the snapshot. Supports https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html[regular expression constructs].
```

Reply:
The trailing dash `-` in the URL needs to be removed. Otherwise, the link breaks in the output.